components/ingest-service/backend/elastic/project_update.go

```diff
@@ -14,38 +13,14 @@ import (
 )
 
 func (backend *Backend) ListProjectUpdateTasks(ctx context.Context) ([]project_update_lib.SerializedProjectUpdateTask, error) {
-	indexNames, err := backend.client.IndexNames()
-	if err != nil {
-		return nil, err
-	}
-
-	tasks := make([]project_update_lib.SerializedProjectUpdateTask, 0, 120)
-	for _, indexName := range indexNames {
-		if strings.HasPrefix(indexName, mappings.Actions.Index) {
-			indexDay, err := mappings.Actions.IndexTimeseriesScanDate(indexName)
-			if err != nil {
-				logrus.WithField("op", "ListProjectUpdateTasks").Warnf("Could not parse date from %s", indexName)
-				continue
-			}
-			priority := indexDay.Unix()
-			params := map[string]string{
-				"n": indexName,
-			}
-			tasks = append(tasks, project_update_lib.SerializedProjectUpdateTask{
-				Priority: priority,
-				Params:   params,
-			})
-		}
-	}
-
-	tasks = append(tasks, project_update_lib.SerializedProjectUpdateTask{
-		Params: map[string]string{
-			"n": mappings.NodeState.Index,
+	return []project_update_lib.SerializedProjectUpdateTask{
+		project_update_lib.SerializedProjectUpdateTask{
+			Params: map[string]string{
+				"n": mappings.NodeState.Index,
+			},
+			Priority: time.Now().Unix(),
 		},
-		Priority: time.Now().Unix(),
-	})
-
-	return tasks, nil
+	}, nil
```

Comment:
With no actions indexes we only need to run the project update on the "node-state" index.